maint.php

```diff
@@ -1054,7 +1056,15 @@ function clearFilter() {
 			form_end_row();
 		}
 	} else {
-		print "<tr><td colspan='8'><em>" . __('No Associated Devices Found', 'maint') . "</em></td></tr>";
+		if ($schedule_created) {
+			if (get_request_var('associated') == 'false') {
+				print "<tr><td colspan='8'><em>" . __('No devices found for selection', 'maint') . "</em></td></tr>";
+			} else {
+				print "<tr><td colspan='8'><em>" . __('No associated devices found for selection.  Uncheck Associated to list unassociated devices.', 'maint') . "</em></td></tr>";
```

Comment:
There is a duplication of the wording with the title, that seems superfluous.

Reply:
This has been changed.  Let me know if I misunderstood.

Reply:
New wording is much clearer, thanks!